package.json

```diff
@@ -58,12 +58,12 @@
   },
```

Comment:
@dudiq maybe make these buttons disabled?
![image](https://user-images.githubusercontent.com/13561530/85127526-41331180-b238-11ea-8e4e-a9eafcbc08c6.png)

Reply:
When I select long video, buttons aren't disabled as well, and I able to upload it
![image](https://user-images.githubusercontent.com/13561530/85127821-d33b1a00-b238-11ea-95e3-e92a6e237619.png)

Reply:
fixed